packages/core/src/WebviewDatadog/__tests__/getInjectedJavaScriptBeforeContentLoaded.test.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-eval */
+import {
+    getInjectedJavaScriptBeforeContentLoaded,
+    DATADOG_MESSAGE_PREFIX
+} from '../__utils__/getInjectedJavaScriptBeforeContentLoaded';
+
+describe('getInjectedJavaScriptBeforeContentLoaded', () => {
+    const DdMessage = 'custom datadog event';
+    let allowedHosts: string[];
+    let script: string;
+    beforeEach(() => {
+        jest.clearAllMocks();
```

Comment:
we should clear `window.DatadogEventBridge` before every test to make sure we don't use the object from the previous `eval` call.
This would also be a good place to set the `postMessage` mock